src/DependencyInjection/Compiler/ExtensionCompilerPass.php

```diff
@@ -51,10 +71,13 @@ public function process(ContainerBuilder $container): void
             }
         }
 
+        /**
+         * @phpstan-var ExtensionMap $extensionConfig
+         */
         $extensionConfig = $container->getParameter('sonata.admin.extension.map');
         $extensionMap = $this->flattenExtensionConfiguration($extensionConfig);
 
-        foreach ($container->findTaggedServiceIds(TaggedAdminInterface::ADMIN_TAG) as $id => $attributes) {
+        foreach ($container->findTaggedServiceIds(TaggedAdminInterface::ADMIN_TAG) as $id => $_attributes) {
```

Comment:
I think this way adds an additional call, but is easier to read.
```suggestion
        foreach (array_keys($container->findTaggedServiceIds(TaggedAdminInterface::ADMIN_TAG)) as $id) {
```

Reply:
I don't really like this. It add one extra calls and maybe some day we will need the tags.
We have some compiler pass which uses tags.

I remove all this unused related changes in order to solve them in another PR.